packages/circularprogress/src/react/__specs__/index.spec.tsx

```diff
@@ -38,6 +38,27 @@ describe('CircularProgress', () => {
     )
   })
 
+  it('displays zero correctly', () => {
+    render(
+      <CircularProgress
+        value={0}
+        aria-label="0% complete"
+        data-testid="zero-test"
+      />
+    )
+
+    // Get the SVG path to test
+    const el = screen
+      .getByTestId('zero-test')
+      .querySelector('.psds-circularprogress__fg')
+
+    // Get the attribute value to test
```

Comment:
Future reference, there's no need for these comments. The code is readable enough. We typically like to roll with using the Clean Code standards - no comments, unless the code is written in a way that requires them - if so, refactor the code to make it cleaner and more readable first...if unable too, leave comment. 👍

Reply:
Thanks, that makes sense.